### DISCUSSION TITLE
Mantine Carousel is not working with autoplay plugin

### DISCUSSION THREAD

**Discussion by EaswarGn** [mantinedev/mantine#6518] [Q&A] [answered]
I was copying the code from the autoplay section here: https://mantine.dev/x/carousel/ 

and I kept getting the error message:
Unhandled Runtime Error
TypeError: embla_carousel__WEBPACK_IMPORTED_MODULE_0__.default.optionsHandler is not a function
 

I already installed all the dependencies with:
npm install @mantine/core @mantine/hooks @mantine/carousel embla-carousel-react

I was trying to use the embla autoplay plugin with the mantine carousel
![image](https://github.com/user-attachments/assets/798b1f27-37b7-4aef-bf12-fc8cb0ef0cea)
pls help 🙏

**Comment by rtivital** [comment]
Add 'use client' to the top of the file.

**Reply by EaswarGn** [reply]
i tried that too, and I still had the same error. I even tried deleting my node_modules folder and running 
npm install
 again, but the error persisted.

**Comment by Willis0826** [comment]
Having the same issue with "@mantine/carousel": "^7.12.2", "embla-carousel-autoplay": "^8.3.0" and "react": "^18"

**Reply by Willis0826** [reply, marked as answer]
Downgrading to "embla-carousel-autoplay": "^7.1.0" fixed the issue
